api/src/main/resources/workbench.yaml

```diff
@@ -470,3 +494,50 @@ definitions:
       phoneNumber:
         description: the user's phone number
         type: string
+
+  CriteriaListResponse:
+    type: "object"
+    required:
+      - "items"
+    properties:
+      items:
+        type: "array"
+        items:
+          $ref: "#/definitions/Criteria"
+
+  Criteria:
+    type: "object"
+    required:
+      - "id"
+      - "type"
+      - "code"
+      - "name"
+      - "group"
+      - "selectable"
+    properties:
+      id:
+        description: id of the criteria
+        type: integer
+        format: int64
+      type:
+        description: type of criteria
+        type: string
+      code:
+        description: what level of data access the user has
+        type: string
+      name:
+        description: description of criteria
+        type: string
+      count:
+        description: est. count in the cdr
+        type: integer
+        format: int64
+      group:
+        description: specifies of child or parent
+        type: boolean
+      selectable:
+        description: specifies if use can search with
```

Comment:
Which criteria would we not be able to search on?

Reply:
see above